pkg/controller/actuator.go

```diff
@@ -353,9 +353,9 @@ func seedObjects(cc *config.ControllerConfiguration, infrastructureConfig *metal
 				"ca.pem":         cc.Accounting.CA,
 				"client.pem":     cc.Accounting.ClientCert,
 				"client-key.pem": cc.Accounting.ClientKey,
+				"d":              "a bug with trailing dashes",
```

Comment:
This is required because of:

https://github.com/gardener/gardener/blob/master/pkg/utils/managedresources/managedresources.go#L488